Grammar Updates to the General Folder (#1026)
AdguardTeam/AdguardKnowledgeBase: 01.general/09.license-key/docs.en.md

<a id="activation_info"></a>
#### Transfering license to another device

> Once the program is activated with a license key, the license key is "bound" to the device on which activation was held. A license key may be activated on a limited number of devices. If the license key is already used on the maximum possible number of devices, you need to reset it before using this key again. You can do it in your [AdGuard Account](#account-functions).

Author: ```suggestion
When you activate the program with a license key, this key gets bound to the device used to perform the activation. The same license key can be activated on a limited number of devices. If the key has already been used on the maximum allowed number of devices, you need to reset it before using this key again. You can do it in your [AdGuard Account](#account-functions).
```

<a id="recovery"></a>
## How to recover my license key?

You can easily find which license keys belong to you in your [AdGuard Account](#account).

The binding of your license key with your account is held with your email address, which was used for the purchase of AdGuard.

**Please note** that it is often not enough to just find out what your license key is. If it was already used on another computer, it’s necessary to unbind it from that computer through the [AdGuard Account](#account) before using it again.

<a id="account"></a>
## How to register the AdGuard Account?

> If you purchased an AdGuard license key, your account is already registered with the email that was used for the purchase. You will receive an email with information about the automatic registration of your personal account, as well as a link to activate it. If you haven’t received this email or forgot your password, you can [recover your password](http://adguard.com/recovery_password.html).

Author: ```suggestion
When you purchase an AdGuard license key, your personal AdGuard account is automatically registered to the email address used for the purchase. An email is sent to this address with the details of the automatic registration of your personal account and an activation link. If for some reason you miss this email or forget your password, you can [recover your password](http://adguard.com/recovery_password.html).
```

This is how you can manage your AdGuard licenses and subscriptions.

#### Renew or upgrade

> You can renew your key for another year, switch to a lifetime protection, or upgrading to the family package (thus adding more devices to it).

Author: ```suggestion
You can renew your key for another year, switch to lifetime protection, or upgrade to a family license (which covers more devices).
```
